### PR TITLE
Use MSIBI class as manager for pair and states

### PR DESCRIPTION
**Pull request by jennyfothergill**
# PR Summary
- Pair and State instances are added to MSIBI instance which handles all communication between them. 
- I removed some options between pair and state. e.g., users can't provide a custom rdf (but I think that would cause more headache than allowing it is worth. The rdf calculation is very fast, and it makes sense to ensure that all rdfs are calculated in the same way.) and alpha form is no longer an option (if we want to add this back it makes sense to have it be a property of the msibi class I think.)
- updated LJ example (working on my end)
- moved rdf function to be method of Pair class

unit tests are not passing yet, but I think it makes sense to wait until we are finished tweaking the format before updating them.

### CODE REVIEW

File: msibi/pair.py [hunk 1]
```diff
@@ -23,113 +20,72 @@ class Pair(object):
...
             self.potential = np.loadtxt(potential)[:, 1]
             # TODO: this could be dangerous
         else:
             self.potential = potential
         self.previous_potential = None
         self.head_correction_form = head_correction_form
 
-    def add_state(
+    def _add_state(
         self,
```
Comment (chrisjonesBSU) [inline msibi/pair.py]: Can we put this on a single line? It would be no where close to 80 columns.

File: msibi/pair.py [hunk 2]
```diff
@@ -23,113 +20,72 @@ class Pair(object):
...
+                self.type1,
+                self.type2,
+                start=-state._opt.max_frames,
+                r_max=state._opt.rdf_cutoff,
+                bins=state._opt.n_rdf_points,
+                exclude_bonded=exclude_bonded
+                )
+        return np.stack((rdf.bin_centers, rdf.rdf*norm)).T
+
     def select_pairs(self, state, exclude_up_to=0):
```
Comment (chrisjonesBSU) [inline msibi/pair.py]: Do we need this function anymore now that we are using freud and bead type names for calcualte RDFs?

**Comment by chrisjonesBSU** [comment]
Looks good. I left a comment for one small change.  The other thing is I'm not sure if we need the `select_pairs` function in `Pair` anymore now that we are using freud for all of the RDFs. I don't think they are used anywhere (with all of the changes we've made) other than a couple unit tests

File: msibi/pair.py (hunk 1 above)
Comment (jennyfothergill) [reply msibi/pair.py]: heck ya

File: msibi/pair.py (hunk 2 above)
Comment (jennyfothergill) [reply msibi/pair.py]: ~select_pairs?~ oops yeah I see it in your below comment. I'll take a look and make sure it's not used anywhere else

**Pull request merged**